client/landing/stepper/declarative-flow/hundred-year-domain-transfer.ts

```diff
@@ -0,0 +1,76 @@
+import { HUNDRED_YEAR_DOMAIN_TRANSFER } from '@automattic/onboarding';
+import { translate } from 'i18n-calypso';
+import { stepsWithRequiredLogin } from 'calypso/landing/stepper/utils/steps-with-required-login';
+import { setSignupCompleteFlowName, setSignupCompleteSlug } from 'calypso/signup/storageUtils';
+import domainTransfer from './domain-transfer';
+import {
+	AssertConditionResult,
+	AssertConditionState,
+	Flow,
+	ProvidedDependencies,
+} from './internals/types';
+
+const hundredYearDomainTransfer: Flow = {
+	...domainTransfer,
```

Comment:
```suggestion
	...domainTransfer,
	name: 'hundred-year-domain-transfer',
```
Could we do this here? Then we wouldn't need to check for the variant slug. But I might be probably missing something 🤔

Reply:
@leonardost The idea of reusing the flow with the onboarding framework is to get all of the features in the parent component - styles included, mainly - and add variants so the child components can differentiate the context. 

We could, indeed, use a different `flowName` instead of the `variantSlug`, but then I think some styles would break.

Would it be OK to deploy this one using the same `flowName` but iterate on it and change it if we feel like it's better? So that we don't block this one because of those checks.